admin-dev/themes/new-theme/scss/pages/_orders_view.scss

```diff
@@ -0,0 +1,98 @@
+#order-view-page {
+  @media not print {
+    .card-details {
+      border: none;
+      box-shadow: none;
+      .card-body {
+        padding: 0;
+      }
+    }
+  }
+
+  > .row:not(.d-print-none) {
+    @media print {
+      display: block !important;
```

Comment:
This is more a question than a comment

Sometimes you put 
/* stylelint-disable declaration-no-important */
And sometimes not
why ?

Reply:
Poke @NeOMakinG

Reply:
Because stylelint is not merged yet.. so anyway I'll have to fix that when I'll rebase it, I think I shouldn't add any stylelint comment at all